src/ui/public/agg_types/__tests__/metrics/percentiles.js

```diff
@@ -0,0 +1,38 @@
+import expect from 'expect.js';
+import ngMock from 'ng_mock';
+import AggTypeMetricPercentilesProvider from 'ui/agg_types/metrics/percentiles';
+import VisProvider from 'ui/vis';
+import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
+
+describe('AggTypeMetricPercentilesProvider class', function () {
+
+  let Vis;
+  let indexPattern;
+  let aggTypeMetricPercentiles;
+
+  beforeEach(ngMock.module('kibana'));
+  beforeEach(ngMock.inject(function (Private) {
+    Vis = Private(VisProvider);
+    indexPattern = Private(FixturesStubbedLogstashIndexPatternProvider);
+    aggTypeMetricPercentiles = Private(AggTypeMetricPercentilesProvider);
+  }));
+
+  it('uses the custom label if it is set', function () {
+    let vis = new Vis(indexPattern, {});
```

Comment:
We should use `const` instead of `let` wherever possible, which is in most places.  The only variables in this file that need `let` are the three at the top of the initial describe.